### PR TITLE
Screen three component created

### PR DESCRIPTION
**Pull request by dreplica**
#### What does this PR do?
It provides an extra component (screenthree.js) on getstarted screen, 

#### Description of Task to be completed?

Have the home screen show the following:

1. a component that displays HairX header.
2.Renders a subtitle and description for client .

#### How should this be manually tested?

After cloning the repo,
`CD` into it and `RUN`

javascript
git checkout develop

git pull origin develop

yarn install

yarn test

Using an emulator: Launch the app, and you should see the home screen after loading.

### CODE REVIEW

File: src/screens/get_started/styles.ts [hunk 1]
```diff
@@ -1,15 +1,53 @@
...
+    top:0px;
+`
+export const Content = styled.View`
+  padding:20px;
+`;
+export const Title = styled.Text`
+  padding:10px 0px;
+  color:orange;
+  font-weight:bold;
+  /* font-size:${({theme})=>theme.fonts.LARGE_SIZE}; */
```
Comment (josemak25) [inline src/screens/get_started/styles.ts:48]: remove all logs or unused codes

File: src/screens/get_started/styles.ts [hunk 2]
```diff
@@ -1,15 +1,53 @@
...
+  font-size:45px;
+`;
```
Comment (josemak25) [inline src/screens/get_started/styles.ts:50]: give a line brake space components are declared too close to each other and make code readability hard

File: src/screens/get_started/styles.ts [hunk 3]
```diff
@@ -1,15 +1,53 @@
...
+  background:${({theme})=>theme.colors.BG_YELLOW_COLOR};
+`
+export const ScreenView = styled.View`
+    position:absolute;
+    width:100%;
+    top:0px;
+`
+export const Content = styled.View`
+  padding:20px;
+`;
```
Comment (josemak25) [inline src/screens/get_started/styles.ts:43]: give a line brake space components are declared too close to each other and make code readability hard

File: src/screens/get_started/styles.ts [hunk 4]
```diff
@@ -1,15 +1,53 @@
...
 
 export const Container = styled.View`
   flex: 1;
+  width:100%;
   justify-content: center;
   align-items: center;
   background-color: ${({ theme }) => theme.colors.BG_LIGHT_COLOR};
 `;
 
+export const Toggle_Nav = styled.View`
```
Comment (josemak25) [inline src/screens/get_started/styles.ts:12]: Stop using camel casing and snake casing together, just use camel casing

File: src/screens/get_started/screenthree.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    Title,
+    Description, 
+    ScreenView
+} from './styles';
+
+const height_for_toggle_buttons = 50;
+const height = (Dimensions.get('window').height) /2
+const content_height = height - height_for_toggle_buttons;
+
+interface iProps {
```
Comment (josemak25) [inline src/screens/get_started/screenthree.tsx:15]: interface created should always start with a capital letter

File: src/screens/get_started/screenthree.tsx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    header:string;
+    title:string;
+    description:string
+}
+
+const ScreenThree:React.FC<iProps> = (props) =>{
```
Comment (josemak25) [inline src/screens/get_started/screenthree.tsx:21]: refactored this code and stop using a react functional component, functional components are mostly used when a component is used as HOC, to pass children to the component, Just use a normal component instead.

File: src/screens/get_started/screenthree.tsx [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+import { Dimensions, ScrollView } from 'react-native';
+import { 
+    Header,
+    Content,
+    Title,
+    Description, 
+    ScreenView
+} from './styles';
+
+const height_for_toggle_buttons = 50;
```
Comment (josemak25) [inline src/screens/get_started/screenthree.tsx:11]: stop using snake casing, use camel casing instead.

If declared variables are constants and won't change on rendering or rerender, like getting the width and height from dimensions, you should name them as capital constant

**EXAMPLE**
```javascript
      const TOGGLE_BUTTON_HEIGHT = 50;

      const HEIGHT = (Dimensions.get('window').height) /2

      const CONTENT_HEIGHT = HEIGHT - TOGGLE_BUTTON_HEIGHT;

```

File: src/screens/get_started/index.tsx [hunk 8]
```diff
@@ -2,17 +2,43 @@ import React from 'react';
...
+        borderWidth:1,width:120,elevation:0}}
+        textStyle={{fontSize:20,fontWeight:"200",color:'black',backgroundColor:'white'}}
+        />
+      </Toggle_Nav>
     </Container>
   );
 }
+
+
+const data = [
```
Comment (josemak25) [inline src/screens/get_started/index.tsx:38]: move this object to a getStarted.json file
and place the JSON file inside libs folder and import it into your screen and use it there.

File: src/screens/get_started/index.tsx [hunk 9]
```diff
@@ -2,17 +2,43 @@ import React from 'react';
...
+      title={item.title}
+      description={item.description}
+      />)}
+      <Toggle_Nav>
+        <Button title={'Skip'}  
+        buttonStyle={{backgroundColor:'white',width:120,elevation:0}}
+        textStyle={{fontSize:20,color:'black',backgroundColor:'white'}}
+        />
+        <Button title={'Next'}  
+        buttonStyle={{backgroundColor:'white',borderColor:'black',
```
Comment (josemak25) [inline src/screens/get_started/index.tsx:28]: use app defined colors, not static colors as defined by you here = "white",

to use app colors, use this function [useThemeContext] by importing and using it.

````javascript

import { useThemeContext } from '../../theme';
import boxShadow from '../../utils/boxShadows';


export default function GetStartedScreen(props: GetStartedScreenProp) {

         const {colors, fonts} = useThemeContext()

  // usuage

          <Button 
               title='Next'
               buttonStyle={{backgroundColor: colors.BG_LIGHT_COLOR, borderColor: colors.BD_DARK_COLOR,
               borderWidth:1, width:120, boxShadow({ elevation: 0}) }}
               textStyle={{fontSize: fonts.LARGE_SIZE, fontWeight: fonts.FONT_WEIGHT_LIGHT, color: colors.FONT_DARK_COLOR, backgroundColor: colors.BG_LIGHT_COLOR}}
        />


// ...rest of your codes
}


```

File: src/router.tsx [hunk 10]
```diff
@@ -14,7 +14,7 @@ export default function AppNavigator() {
   return (
     <NavigationContainer>
       <RootStack.Navigator
-        initialRouteName="SplashScreen"
+        initialRouteName="GetStartedScreen"
```
Comment (josemak25) [inline src/router.tsx:17]: after you finish testing rename the initial route back to SplashScreen, as to help reduce merge conflict with others PR

File: src/screens/get_started/index.tsx [hunk 11]
```diff
@@ -2,17 +2,43 @@ import React from 'react';
 import Button from '../../components/button';
 import { NavigationInterface } from '../types';
 
-import { Container, Welcome } from './styles';
+import { Container,Toggle_Nav } from './styles';
```
Comment (ogbodo) [inline src/screens/get_started/index.tsx]:
```suggestion
import { Container,ToggleNav } from './styles';
```

Comment (ogbodo) [inline src/screens/get_started/index.tsx]: Please stick the PascalCase naming convention when naming your components

Comment (ogbodo) [inline src/screens/get_started/index.tsx]:
```suggestion
import { Container,ToggleNav } from './styles';
```
Please stick the PascalCase naming convention when naming your components

File: src/screens/get_started/index.tsx [hunk 12]
```diff
@@ -2,17 +2,43 @@ import React from 'react';
...
-      <Button title="Get started screen button" />
-      <Welcome>Get Started Screen</Welcome>
+      {data.map((item,index)=><ScreenThree 
+      key={index} 
+      header={item.header}
+      title={item.title}
+      description={item.description}
+      />)}
+      <Toggle_Nav>
+        <Button title={'Skip'}
```
Comment (ogbodo) [inline src/screens/get_started/index.tsx]:
```suggestion
        <Button title='Skip'
```
You could actually do it like this since its not a JS variable

Comment (dreplica) [reply src/screens/get_started/index.tsx]: they are from the branch i merged, Amakri corrected me on that yesterday

File: src/screens/get_started/index.tsx (hunk 11 above)
Comment (dreplica) [reply src/screens/get_started/index.tsx]: they are from the branch i merged, Amakri corrected me on that yesterday

Comment (dreplica) [reply src/screens/get_started/index.tsx]: wait you are viewing an old PR, i updated those since yesterday

**Pull request merged**